beacon_node/beacon_chain/src/beacon_block_reward.rs

```diff
@@ -0,0 +1,55 @@
+use crate::{BeaconChain, BeaconChainError, BeaconChainTypes};
```

Comment:
Lets delete this file until it is added in #3907. Otherwise we'll get merge conflicts once this PR is merged